lib/pure/uri.nim

```diff
@@ -255,18 +307,13 @@ proc combine*(base: Uri, reference: Uri): Uri =
   ## URI's path affect the resulting URI.
   ##
   ## For building URIs you may wish to use \`/\` instead.
```

Comment:
`\` should be a link to that proc:

```
For building URIs you may wish to use `/ proc <#/,Uri,string>`_ instead.
```

Reply:
72ee0b8d6668d76073c11f8b86bb54772c248aff